source/bot.py

```diff
@@ -32,15 +32,21 @@ async def download(self, url: str, message: Message) -> io.BytesIO:
         """
         async with aiohttp.ClientSession(raise_for_status=True) as session:
             async with session.get(url) as resp:
-                size = int(resp.headers.get("Content-Length", 0))
+                size = int(resp.headers.get("Content-Length", "0"))
```

Comment:
Please revert. `int(0)` should be _slightly_ more efficient than `int("0")`, and python is dynamically typed so it doesn't care.